### ISSUE TITLE
French Localization

### ISSUE THREAD

**Issue by quentinglorieux** [sveltia/sveltia-cms#84]
I am working on french localization. I will propose a PR as soon as it is stabilized.

**Comment by kriek** [comment]
Hello, I can also offer my support on this when sveltia-cms is ready.  Translating or reviewing.

**Comment by aloxe** [comment]
@quentinglorieux @kriek count mein too.

**Comment by EKliptus** [comment]
Please, when ready for translation ? What is the roadmap about translation of cms back office ?

**Comment by kyoshino** [comment]
Before the 1.0 string freeze, we have to complete these things mentioned on [our roadmap](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#roadmap), which are actually dozens of items:

> Enhanced [compatibility with Netlify/Decap CMS](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#current-limitations)
> Tackling some more Netlify/Decap CMS issues
> Accessibility audit

 So l10n won’t be ready until this Q4.

**Comment by kyoshino** [comment]
Sorry for keeping you waiting for so long! Sveltia CMS is now ready for localization. We’ll be adding more strings before and after the 1.0 release, but it’s a good time to start translating. Check out the [README](https://github.com/sveltia/sveltia-cms/blob/main/src/lib/locales/README.md) for instructions. Let me know if you have any questions!

**Comment by ThibaudMZN** [comment]
@quentinglorieux @kriek @kyoshino cout me in too for this translation !

**Comment by kyoshino** [comment]
@quentinglorieux Can you still work on this?

**Comment by rubendossantos** [comment]
Hi @kyoshino, I'd be happy to help with the French localization once the sveltia-i18n / MessageFormat 2 migration lands and the strings are ready for translation. I'm a fluent French speaker and I use Sveltia CMS in production for client sites in French-speaking markets (Switzerland, France, Luxembourg). Feel free to ping me here when contributions open up.

**Comment by kyoshino** [comment]
Let me ping some folks who have already shown up here...

@kriek @aloxe @EKliptus Can anyone help with French localization?

**Comment by bnjbvr** [comment]
I'd be happy to help too, fwiw!

**Comment by kriek** [comment]
Hi @kyoshino ,
on my side, I miss time in August but I can work on it beginning of September.

**Comment by aloxe** [comment]
Thanks for pinging me @kyoshino, I am still available to help translate in French but I won't have time for this before mid August.

**Comment by kyoshino** [comment]
Thank you, folks!

@ThibaudMZN Do you have time to work on this now?

**Comment by tomgue** [comment]
I've started translating Sveltia UI, I can submit a pull request in draft while I translate the CMS part.

**Comment by kyoshino** [comment]
@tomgue Please read the [README](https://github.com/sveltia/sveltia-cms/blob/main/src/lib/locales/README.md):

> To prevent unnecessary duplication of effort, PRs that are not linked to an issue or sent by unassigned people will not be accepted.

**Comment by kyoshino** [comment]
I’d ping folks one by one… @rubendossantos Do you have the bandwidth available right now?

**Comment by rubendossantos** [comment]
Yes — I'll get both PRs (sveltia-cms + sveltia-ui) up today.

**Comment by rubendossantos** [comment]
Done — sveltia/sveltia-cms#893 and sveltia/sveltia-ui#19 are up, translated from the current en-US HEAD (includes the 24 keys added since the README's pinned commit).

**Comment by kyoshino** [comment]
French shipped with [v0.187.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.187.0) 🇫🇷 Thanks, @rubendossantos, for the quick work!

**Comment by EKliptus** [comment]
> French shipped with [v0.187.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.187.0) 🇫🇷 Thanks, [@rubendossantos](https://github.com/rubendossantos), for the quick work!

@rubendossantos and others...
Thanks a lot for the quick work! Sorry, I didn’t even have time to reply before it was already done. Really appreciate it!

**Comment by kyoshino** [comment]
@rubendossantos Hi there! I just synced the latest English strings. To save you time, I had Claude translate the new ones for you. Please review the translations and update them if necessary. [Here’s the diff](https://github.com/sveltia/sveltia-cms/commit/a0a0851d05bf79298e5ba824a14b6bad7708f871#diff-49c5db60311dabc0eaa4d6f87b092633be7f16b0286faedbc93bb5a90d73c2a1).

**Comment by rubendossantos** [comment]
Thanks for the heads-up and for pre-translating the new strings. Reviewed and corrected in #971, now merged. Only one fr string needed rewording; the rest of the sync was fine.

**Comment by kyoshino** [comment]
The fix shipped with [v0.211.3](https://github.com/sveltia/sveltia-cms/releases/tag/v0.211.3).
